spa/src/components/dashboard/connectStripe/userHasSingleRPNotConnectedToStripe.js

```diff
@@ -0,0 +1,15 @@
+function userHasSingleRPNotConnectedToStripe(userDetails) {
+  if (userDetails && userDetails.revenue_programs && userDetails.revenue_programs.length === 1) {
+    // Add a check for the RP Payment Provider when api returns
```

Comment:
Add `TODO:`, there are some toold in VS Code that identifies all `TODO:`s in project. It helps to know where changes need to be done 
```suggestion
    // TODO: Add a check for the RP Payment Provider when api returns
```